migrations/1601022564938-copy-parents-data-into-children-entities-and-delete-parent-users.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-await-in-loop */
+// eslint-disable-next-line no-unused-vars
+const mongoose = require('mongoose');
+
+const { info } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+const parentSchema = new mongoose.Schema({
+	firstName: { type: String, required: true },
+	lastName: { type: String, required: true },
+	email: { type: String, required: true, lowercase: true },
+});
+
+const consentForm = ['analog', 'digital', 'update'];
+
+const OldUser = mongoose.model(
+	'oldUser',
```

Comment:
AFAIK, the model name has to be unique. I would propose to add a date statement or some random number to avoid conflicts e.g. oldUser_05102020. The same applies to other models

Reply:
done